Lib/test/test_binascii.py

```diff
@@ -159,6 +162,9 @@ def assertDiscontinuousPadding(data, non_strict_mode_expected_result: bytes):
         assertLeadingPadding(b'===', b'')
         assertDiscontinuousPadding(b'ab=c=', b'i\xb7')
         assertDiscontinuousPadding(b'ab=ab==', b'i\xb6\x9b')
+        assertIncorrectPadding(b'AAAA=', b'\x00\x00\x00')
```

Comment:
Leading and discontinuous padding are also 'incorrect'.  'ExcessPadding' (at the end) seems more specific.

Reply:
I agree that `Excess padding` describes the problem more accurately, and it is more helpful for people who are not familiar with base64.

I used `Incorrect padding` just to avoid introducing another exception message (so that it won't break anything).

Both are OK for me.